src/bindings/javascript/generator.cpp

```diff
@@ -30,5 +32,7 @@ EMSCRIPTEN_BINDINGS(libcellml_generator)
         .function("setModel", &libcellml::Generator::setModel)
         .function("interfaceCode", &libcellml::Generator::interfaceCode)
         .function("implementationCode", &libcellml::Generator::implementationCode)
+        .class_function("equationCodeUsingDefaultProfile", select_overload<std::string(const libcellml::AnalyserEquationAstPtr &)>(&libcellml::Generator::equationCode))
+        .class_function("equationCodeUsingGivenProfile", select_overload<std::string(const libcellml::AnalyserEquationAstPtr &, const libcellml::GeneratorProfilePtr &)>(&libcellml::Generator::equationCode))
```

Comment:
The way we have been naming these overloaded methods is to use the name of the first parameter in the originating C++ method. These two methods would then become:
```
equationCode
```
and
```
equationCodeByProfile
```

Reply:
a291966e